src/views/SystemIntake/ContractDetails/index.tsx

```diff
@@ -154,6 +154,29 @@ const ContractDetails = ({ systemIntake }: ContractDetailsProps) => {
   };
 
   const onSubmit = (values?: ContractDetailsForm) => {
+    const intakePayload = { ...values };
+
+    if (
+      values?.contract.hasContract === 'NOT_STARTED' ||
+      values?.contract.hasContract === 'NOT_NEEDED'
+    ) {
+      intakePayload!.contract = {
```

Comment:
wow! the `!` here is new to me - that is cool

Reply:
Normally you should only use that when you know typescript can't discern it's value.  The conditional above would only hit that line if `values?.contract` existed, but TS doesn't seem to be able to decipher that